specification/inference/inference/InferenceRequest.ts

```diff
@@ -54,7 +54,7 @@ export interface Request extends RequestBase {
      */
     query?: string
     /**
-     * Text input to the model.
+     * Inference input
```

Comment:
Adding a full stop here will clarify the one-line documentation in Python code.

```suggestion
     * Inference input.
```